applications/openshift/risk-assessment/container_security_operator_exists/rule.yml

```diff
@@ -29,6 +29,7 @@ rationale: |-
 
 references:
   nist: CM-6(b)
+  pcidss: Req-8.6.1,Req-8.6.2,Req-8.6.3
```

Comment:
The `pcidss` references are for PCI-DSS 3.2.1.
The key for PCI-DSS 4.0 references is `pcidss4`.

But the references can be added through the control file, rather than though each rule.

Reply:
I think that makes sense, I think this rule should only apply to control `6.3.2`  the other 3 controls should be met by rule like `accounts-unique-service-account`

Reply:
@Vincent056 Do you plan to change the references?

Reply:
sorry the change was not pushed, just updated it now